public/components/explorer/explorer.tsx

```diff
@@ -113,18 +116,21 @@ export const Explorer = ({
     [setFixedScrollEl]
   );
 
-  const getIndexFromRawQuery = (rawQuery: string) => rawQuery.split('=')[1].split(' ')[0];
-
-  useEffect(() => {
+  const fetchData = () => {
     if (!query) return;
-    if (query.includes('stats')) {
-      const index = getIndexFromRawQuery(query);
+    if (statsRegx.test(query)) {
```

Comment:
similar here, this is the same as `query.includes('stats')`. maybe check if `stats` is followed directly (or with some spaces) after `|`, or some other conditions that's only true for the `stats` command?

Reply:
Sure, I'm thinking to use query.match(/\|\s*stats/) here to match 'stats'.

Reply:
yea, makes sense, I'm thinking and testing query.match(/\|\s*stats/i) to match 'stats'

Reply:
cool, i think the `|` character might need escape

Reply:
yes, the one I used includes the escape, I noticed that when I left the comment it was missing that escape. I tried to modified it to add escape but it just not reflecting it. Wired.